File: exercice2/scala/src/main/scala/fr/devoxx/devops/logs/spark/Spark2.scala
package fr.devoxx.devops.logs.spark

import fr.devoxx.devops.logs.ApacheAccessLog
import org.apache.spark.rdd.RDD

/* Répartition des codes http */
case class Spark2(rdd: RDD[String]) {

  def process: Array[(Int, Long)] = {
    Array((0, 0L))
  }
}

File: exercice2/java/src/main/java/fr/devoxx/devops/logs/spark/Spark2.java
package fr.devoxx.devops.logs.spark;

import fr.devoxx.devops.logs.ApacheAccessLog;
import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaRDD;
import org.apache.spark.api.java.JavaSparkContext;
import scala.Tuple2;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

/* Répartition des codes http */
public class Spark2 implements Serializable {

    public List<Tuple2<Integer, Long>> process(JavaRDD<String> rdd) {
        return new ArrayList<>();
    }

}
